modules/issue_tracker/jsx/issueIndex.js

```diff
@@ -474,24 +474,82 @@ class CollapsibleComment extends React.Component {
       "Show Comment History" :
       "Hide Comment History");
 
+    const hrStyle = {
+      marginTop: "8px",
```

Comment:
Since you are revamping this, I think it would make sense to create an `IssueHistory` component that will do all that stuff below